.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: ruby
 rvm:
  - 2.3.3
-script: bundle exec rake jasmine:ci
+script:
+  - bundle exec rake jasmine:ci
+  - gem install middleman
```

Comment:
It'd be nice to roll this up into something – possibly a rake task? There's a lot of stuff going on in here and it's not immediately obvious that the aim is to test the thing.